### PR TITLE
Update dev_box

### PR DESCRIPTION
**Pull request by crutkas**
Updating to 
- UWP Workload
- Azure workload
- Windows Template Studio 
- disabling hyper-v and WSL since choco would throw errors

### CODE REVIEW

**Comment by yodurr** [comment]
Thanks Clint!  I'll test it.  If more people can try this as well and provide feedback that is much appreciated.

**Comment by crutkas** [comment]
i do believe we should add Code back in and i would like to do for future scripts break stuff apart so you can pick and choose.

**Comment by yodurr** [comment]
Would also like to see common scripts so you can pick an choose.  Otherwise we're going to have a lot of duplication across this repo.
https://github.com/Microsoft/windows-dev-box-setup-scripts/issues/2

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To run a setup script, click a link in the table below from your target machine.
 
 |Click link to run  |Description  |
 |---------|---------|
-|<a href='http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/Microsoft/windows-dev-box-setup-scripts/master/dev_app.ps1'>Desktop App</a>     | Windows Desktop App Development (Visual Studio, Windows SDK) |
+|<a href='http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/crutkas/windows-dev-box-setup-scripts/master/dev_app.ps1'>Desktop App</a>     | Windows Desktop App Development (Visual Studio, Windows SDK) |
```
Comment (yodurr) [inline README.md]: please undo this change :)

File: dev_app.ps1 [hunk 2]
```diff
@@ -4,26 +4,30 @@
 
 Disable-UAC
 
-#--- Windows Features ---
-Set-WindowsExplorerOptions -EnableShowHiddenFilesFoldersDrives -EnableShowProtectedOSFiles -EnableShowFileExtensions
+invoke-expression -ConnectionUri https://raw.githubusercontent.com/crutkas/windows-dev-box-setup-scripts/master/helper_WindowsExplorerSettings.ps1
```
Comment (yodurr) [inline dev_app.ps1]: will need to change crutkas to windows.  

When I run it I get "ERROR: A parameter cannot be found that matches parameter name 'ConnectionUri'."

**Comment by crutkas** [comment]
damn it, if continue to do stuff, it looks like it is pulling that diff

**Comment by crutkas** [comment]
reset this now ....
